ISISComputingGroup/ibex_gui
# Only delete key now removes PVs

### Description of work

See https://github.com/ISISComputingGroup/IBEX/issues/1777

### To test

* Go to the PV values tab in the edit config dialog.
* Select a PV. Press any key and confirm that the PV is not deleted.
* Select a PV. Press the delete key and confirm that the PV is deleted. 

---

#### Code Review


### Functional Tests


### Final Steps
